src/main/resources/workerScripts/fromMultiplePriorityQueues.lua

```diff
@@ -15,8 +18,9 @@ for q in queues.gmatch(queues, NEXT_QUEUE_REGEX) do
     local queueName = 'resque:queue:' .. q
     local status, queueType = next(redis.call('TYPE', queueName))
     local payload
+
     if queueType == 'zset' then
-        local firstMsg = redis.call('ZRANGE', queueName, '0', '0')
+        local firstMsg = redis.call('ZRANGEBYSCORE', queueName, '-inf', now, 'WITHSCORES', 'LIMIT', '0', '1')
```

Comment:
Maybe I forgot already, but why `WITHSCORES` is needed here?

Reply:
that's not needed indeed, nice catch

i also removed it from the `jesque_pop` lua script